deploy/examples/create-external-cluster-resources.py

```diff
@@ -969,6 +970,16 @@ def get_rbd_node_caps_and_entity(self):
 
         return caps, entity
 
+    def get_healthchecker_caps_and_entity(self):
+        entity = "client.healthchecker"
+        caps = {
+            "mgr": "allow command config",
```

Comment:
this sequence is changed that's why unit test are not able to wok,

`mon` cap was above before

Reply:
okay, I have updated but it's weird to fail test on that

Reply:
Actually the values above are hardcoded so we expect the values in the same order